mm2src/coins/eth.rs

```diff
@@ -796,6 +813,296 @@ async fn withdraw_impl(coin: EthCoin, req: WithdrawRequest) -> WithdrawResult {
     })
 }
 
+/// `get_nft_list` function returns list of NFTs on ETH or/and BNB chains owned by user.
+pub async fn get_nft_list(ctx: MmArc, req: NftListReq) -> MmResult<NftList, GetNftInfoError> {
```

Comment:
Nft and moralis implementations should move to outside of `eth` module. This module already has way too much logic in it. Having too long source files impacts incremental compilations in a bad way.

Reply:
Yea, already did it will push later.

Reply:
`get_nft_list`, `get_nft_metadata`, `get_nft_transfers`, `withdraw_nft` with `send_moralis_request` are in nft module.
I left `withdraw_erc721` and `withdraw_erc1155` in `eth`